.github/workflows/test-future.yml

```diff
@@ -31,3 +31,10 @@ jobs:
       - name: Test with pytest
         run: |
           pytest
+      - name: Note a possible Black incompatibility and required actions
+        if: failure()
+        run: |
+          echo "### :x: Black incompatibility? :x: " >>$GITHUB_STEP_SUMMARY
+          echo "" >>$GITHUB_STEP_SUMMARY
+          echo "See [#382](/akaihola/darker/issues/382)" >>$GITHUB_STEP_SUMMARY
+          echo "for possible required actions" >> $GITHUB_STEP_SUMMARY
```

Comment:
How to best test this?

Reply:
Probably simply by introducing a deliberate test failure, and removing it before merging.

Reply:
I now added a failing test in `test_black_diff.py`.

Reply:
@akaihola Yes, you could do that. Maybe do it on another branch to prevent "force-push" pollution of this PR.

Reply:
Oops, I already did it here. But I'm anyway doing quite a bit of force-pushing when working on branches, so it doesn't make much of a difference.

Reply:
There is now a failing test build [here](https://github.com/akaihola/darker/actions/runs/3782380630/jobs/6430076182), but I can't find the message anywhere.

Reply:
It is the `build and test` workflow, not the `test future` workflow. That seems to succeed.

Reply:
...ah, of course, since `test-future` is only run nightly and not within PRs. We'll have to wait.

```yaml
name: test-future
on:
  schedule:
    - cron: "05 20 * * 6"
```

Also, I guess it would be possible to show a message embedded in the "Files changed" view on the line where we would then need to insert the maximum version constraint for Black.

Reply:
Perhaps like this:
```bash
LINENUM=`sed -n '/black>=/=' setup.cfg|head -1
echo "::notice file=setup.cfg,line=${LINENUM},title=Add Black constraint::You could add a maximum version constraint for Black here"
```

With more magic, we could even find out the latest Black release version number, show `,<=X.Y.Z` and point to the correct column to insert that into...

I guess a code suggestion (as available in review comments) isn't possible, we would need a bot for that.

Reply:
I added that and temporarily enabled `test-future` in the PR so we can see the result sooner.

Reply:
The notice message worked correctly as can be seen in "Files changed" currently:
![kuva](https://user-images.githubusercontent.com/13725/209571351-84e659f4-32b1-44b5-a767-6cc438174055.png)

Reply:
>With more magic, we could even find out the latest Black release version number, show ,<=X.Y.Z and point to the correct column to insert that into...

I added this as well.

Reply:
And the job summary is visible on the [summary page of the `test-future` workflow run](https://github.com/akaihola/darker/actions/runs/3782833008):
![kuva](https://user-images.githubusercontent.com/13725/209575322-33403cf8-971c-42d4-90b6-e26bc1d8dfdb.png)

Reply:
The intentional test failure and the temporary modifications to `test-future.yml` have now been reverted. I'll rebase later to hide them from history as well.

Reply:
Rebased.